extensions/sql-migration/src/constants/strings.ts

```diff
@@ -761,6 +833,9 @@ export function WARNINGS_COUNT(totalCount: number): string {
 export const AUTHENTICATION_TYPE = localize('sql.migration.authentication.type', "Authentication type");
 
 export const REFRESH_BUTTON_LABEL = localize('sql.migration.status.refresh.label', 'Refresh');
+export const STATUS_LABEL = localize('sql.migration.status.status.label', 'Status');
+export const SORT_LABEL = localize('sql.migration.migration.list.sort.label', 'Sort');
+export const ASSENDING_LABEL = localize('sql.migration.migration.list.assending.label', 'Assending');
```

Comment:
typo, I'm assuming you meant 'Ascending' here

Reply:
yes, thank you for catching that